src/templates/template-vision-classification/main.py

```diff
@@ -109,15 +125,32 @@ def _():
             exp_logger.finish()
         elif exp_logger:
             exp_logger.close()
+    #::: } :::#
 
 
-# main #
+# main
 def main():
-    parser = ArgumentParser(parents=[get_default_parser()])
+    config = OmegaConf.load("./config.yaml")
```

Comment:
Went with simple `OmegaConf.load` -> dict -> Namespace flow.

Reply:
@ydcjeff i think if we are using basic configs without creating object from the config, we can reuse PyYAML instead of OmegaConf. As an option we could use Hydra as more sophisticated config manager